services/notification_service.py

```diff
@@ -0,0 +1,43 @@
+import logging
+import os
+import firebase_admin
+from firebase_admin import credentials, messaging, exceptions
+from typing import List, Optional
+
+
+class NotificationService:
+
+    def __init__(self):
+
+        if not firebase_admin._apps:
+            cred_path = os.getenv('GOOGLE_APPLICATION_CREDENTIALS', r"C:\Users\User\backend\google-credentials.json")
```

Comment:
Please just have the path always be the root of the repository. It shouldn't be necessary to hardcode your path like you've done here, you should be able to call "os.getcwd()" like `f"{os.getcwd()}/google-credentials.json"`